frontend/src/scenes/notebooks/NotebookPanel/notebookPopoverLogic.ts

```diff
@@ -127,31 +30,6 @@ export const notebookPopoverLogic = kea<notebookPopoverLogicType>([
         },
     }),
 
-    listeners(({ cache, actions, values }) => ({
-        startDropMode: () => {
-            cache.dragEntercount = 0
-            cache.dragStart = null
-            actions.setPopoverVisibility('peek')
-
-            cache.dragListener = (event: MouseEvent) => {
-                if (!cache.dragStart) {
-                    cache.dragStart = event.pageX
-                }
-
-                // The drop distance is the percentage between where the drag started and where it now is
-                const dropDistance = (event.pageX - cache.dragStart) / window.innerWidth
-                actions.setDropDistance(dropDistance)
-            }
-            window.addEventListener('drag', cache.dragListener)
-        },
-        endDropMode: () => {
-            if (values.popoverVisibility === 'peek') {
-                actions.setPopoverVisibility('hidden')
-            }
-            window.removeEventListener('drag', cache.dragListener)
-        },
-    })),
-
     urlToAction(({ actions, values }) => ({
```

Comment:
This can go too. It serves to close the notebook popover if you click a link (in a notebook) that brings you to a different URL in PostHog. Not a problem in the new UI because the side panel can stay open alongside content